Magento_Cms/web/js/custom-country-form.js

```diff
@@ -0,0 +1,66 @@
+define(['jquery', 'mage/url', 'jquery-ui-modules/widget', 'jquery/validate'], (
+  $,
+  urlBuilder
+) => {
+  $.widget('vaimo.countryFormWidget', {
+    options: {},
+
+    _create() {
+      this.submitForm();
+    },
+
+    submitForm() {
+      const self = this;
+
+      $('#search-country-form').on('submit', function (e) {
```

Comment:
if the script is already called for the element you always able to access it via
```js
this.element
```
don't try to find it via querySelector